Common/ML/include/ML/3rdparty/GPUORTFloat16.h

```diff
@@ -9,6 +9,7 @@
 #include <cmath>
```

Comment:
Headers that go to the GPU must not include and C/C++ Std header.
Please protect them with
```
#ifndef GPUCA_GPUCODE_DEVICE
#include ...
#endif
```
And instead of <stdint.h> you should use <cstding>